src/server/services/user.service.ts

```diff
@@ -543,13 +543,13 @@ export const removeAllContent = async ({ id }: { id: number }) => {
     select: { id: true },
   });
 
-  const res = await dbWrite.$transaction([
+  const res = await Promise.all([
```

Comment:
I think this would make a new connection for each query, filling up our pool of connection probably. I might be wrong. Also, if we don't make them in a transaction and one of them fails, it might leave things in an incomplete state 😬